client/src/components/PageExplorer/PageExplorer.scss

```diff
@@ -61,10 +61,14 @@ $explorer-header-horizontal-padding: 10px;
 .c-page-explorer__header__title__inner {
   @apply w-flex;
   padding: 1em $explorer-header-horizontal-padding;
+  padding-bottom: 1rem;
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
 
+  outline: 1px solid #24184d;
```

Comment:
I see why you added this now, but if you think about the odd '2px' above needed (why you had 48px and not 50px) it is because of this outline added to the content.

The problem with outline as it does not add to the box model (size) of the container, so it was just adding a 'fake' border around it to it looked like it it fit but it was not the actual container div.

on top of that, using outline to always show a fake border is not a good idea as it makes accessibility harder.

https://developer.mozilla.org/en-US/docs/Web/CSS/outline

One thing that makes Wagtail a bit more confusing is that the focus outline (a yellow ish 3px outline) only shows when the keyboard is active.

```suggestion
```